core/src/main/java/com/cloud/serializer/GsonHelper.java

```diff
@@ -51,6 +51,8 @@ public class GsonHelper {
         GsonBuilder loggerBuilder = new GsonBuilder();
         loggerBuilder.disableHtmlEscaping();
         loggerBuilder.setExclusionStrategies(new LoggingExclusionStrategy(s_logger));
+        loggerBuilder.serializeSpecialFloatingPointValues();
+        // maybe add loggerBuilder.serializeNulls(); as well?
```

Comment:
add or not ?

Reply:
no idea, I think we can leave it for now

Reply:
yes, I was giving as an hint/option to custom builders.

Reply:
ok, fair